permits/forms.py

```diff
@@ -828,7 +828,11 @@ class PermitRequestAdditionalInformationForm(forms.ModelForm):
 
     class Meta:
         model = models.PermitRequest
-        fields = ["is_public", "status"]
+        fields = [
+            "is_public",
+            "shortname",
```

Comment:
Hide this field if `ENABLE_GEOCALENDAR` is false (like is_public)

Reply:
Sure ? A short name ight be helpful anyway IMHO

Reply:
> Sera affiché dans le calendrier si la demande est rendue tout publique

It doen't make sense if no calendar...

Reply:
Maybe in the list view or in the printed pdf  ?